### PR TITLE
Failed TC

### PR DESCRIPTION
**Pull request by JansiSasiumar12**

### CODE REVIEW

File: panda-qa/Resources/DefinedKeywords.robot
Comment (ArchanaSaini149) [inline panda-qa/Resources/DefinedKeywords.robot:1]: We dont need TC for Table alignment. That was a manual task. Please remove this code form Automation.

File: panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot
Comment (ArchanaSaini149) [inline panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot:1]: Remove "r" from first line.

Comment (ArchanaSaini149) [inline panda-qa/Resources/PageObjects/BarronsWatchlistPage.robot:1]: remove method "Validate Table Alignment". We dont need automation for table alignment

File: panda-qa/Tests/FollowWidget/Regression/Barrons_Regression.robot
Comment (ArchanaSaini149) [inline panda-qa/Tests/FollowWidget/Regression/Barrons_Regression.robot:1]: remove automation for #US-4049. We dont need automation for Table alignment

**Review by ArchanaSaini149 — CHANGES_REQUESTED** [review]
Please review the comments, fix them and re-submit. Thank you !

**Comment by JansiSasiumar12** [comment]
OK. I will correct and submitted

**Comment by JansiSasiumar12** [comment]
Resubmitted

**Pull request closed without merging**